node/internal/primordials.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 20222 the Deno authors. All rights reserved. MIT license.
+
+export const ArrayIsArray = Array.isArray;
+export const ArrayPrototypeForEach = (that, ...args) => that.forEach(...args);
```

Comment:
I guess `ArrayPrototypeForEach = Array.prototype.forEach` might be more straightforward?